tests/typecheck/managers/querysets/test_values_list.yml

```
@@ -322,3 +322,21 @@
                 class Blog(models.Model):
                     num_posts = models.IntegerField()
                     text = models.CharField(max_length=100, blank=True)
+-   case: handles_field_with_same_name_on_other_model
+    main: |
+        from myapp.models import A
+        reveal_type(A.objects.values_list("b__name").get())  # N: Revealed type is "Tuple[builtins.str]"
```

Let's use two names just to be sure they are different.

```suggestion
        reveal_type(A.objects.values_list("name", "b__name").get())  # N: Revealed type is "Tuple[builtins.str]"
```

---

good call!